### PR TITLE
Update f-droid to v1.2.3

### LINKED ISSUE #122: Update the f-droid branch to v1.2.3

### PR DESCRIPTION
**Pull request by bassosimone**
Closes #122

### CODE REVIEW

**Comment by lorenzoPrimi** [comment]
🐬

File: app/build.gradle [hunk 1]
```diff
@@ -29,25 +27,17 @@ android {
...
-//ALT: https://developers.google.com/instance-id/guides/android-implementation
+
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile('com.crashlytics.sdk.android:crashlytics:2.6.6@aar') {
-        transitive = true;
-    }
-    //compile 'com.android.support:customtabs:25.3.1'
-    compile 'org.openobservatory.measurement_kit:android-libs:0.7.4-1'
+    compile 'org.openobservatory.measurement_kit:android-libs:0.7.6-1'
```
Comment (lorenzoPrimi) [inline app/build.gradle]: Was it released?

Comment (bassosimone) [reply app/build.gradle]: Just now :-D

**Comment by bassosimone** [comment]
@lorenzoPrimi thanks for f89344a! I am now going to write changelog, tag, and merge!

**Pull request merged**